exercises/practice/circular-buffer/.docs/instructions.append.md

```diff
@@ -0,0 +1,6 @@
+# Set `errno` to a special value
+If the buffer is empty during a read attempt, an error is raised to alert the client.
+
+These error values can be found in the `errno.h` header file.
+
+[Wikipedia](https://en.wikipedia.org/wiki/Errno.h)
```

Comment:
```suggestion
These error values can be found in the [`errno.h`][errno.h] header file.

[errno.h]: https://en.wikipedia.org/wiki/Errno.h
```

This switches the link to be inline. This is hopefully a bit more usable and is consistent with how we've handled similar links elsewhere in the track.